app/components/Checkbox.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Image, StyleSheet, TouchableOpacity } from 'react-native';
+
+import boxCheckedIcon from './../assets/images/boxCheckedIcon.png';
+import boxUncheckedIcon from './../assets/images/boxUncheckedIcon.png';
+import Colors from '../constants/colors';
+import { Typography } from './Typography';
+
+export const Checkbox = props => {
+  const { t } = useTranslation();
+
+  return (
+    <TouchableOpacity
+      style={{ flexDirection: 'row' }}
+      onPress={props.onPressFunction}>
+      <Image
+        source={props.boxChecked === true ? boxCheckedIcon : boxUncheckedIcon}
+        style={{ width: 25, height: 25, marginRight: 10 }}
+      />
+      <Typography style={styles.checkboxText}>
+        {t('onboarding.eula_checkbox')}
+      </Typography>
+    </TouchableOpacity>
+  );
+};
+
+const styles = StyleSheet.create({
+  checkboxText: {
+    color: Colors.WHITE,
```

Comment:
color is set by outer wrapped Theme

Font size 18 is set by typography use = x

These can be removed